tests/e2e/lib.sh

```diff
@@ -292,6 +292,8 @@ deploy_sensor_via_operator() {
 }
 
 export_central_basic_auth_creds() {
+    DEPLOY_DIR="deploy/${ORCHESTRATOR_FLAVOR}"
```

Comment:
This `DEPLOY_DIR` variable has been a pain and source of bugs since ACS moved off Circle CI and I'm not clear as to the best way to deal with it but all that aside, probably only set in this function if not already set.

```suggestion
    DEPLOY_DIR="${DEPLOY_DIR:-deploy/${ORCHESTRATOR_FLAVOR}}"
```